### ISSUE TITLE
Null character in friendly_name on sensors

### ISSUE THREAD

**Issue by tedenda** [angoyd/freshintelliventHacs#8]
For some reason there is some null character in friendly_name on sensors from this integration.
Friendly name is also in " ", I see other sensors have the name without " ", I don't know the difference.

Current speed sensor:
```
state_class: measurement
unit_of_measurement: rpm
friendly_name: "Lilla toan\0\0\0\0\0\0\0\0\0\0 Current speed"
```

Humidity sensor
```
state_class: measurement
unit_of_measurement: '%'
device_class: humidity
friendly_name: "Lilla toan\0\0\0\0\0\0\0\0\0\0 Humidity"
```

Mode sensor
```
state_class: measurement
friendly_name: "Lilla toan\0\0\0\0\0\0\0\0\0\0 Mode"
```

Mode raw sensor
```
state_class: measurement
friendly_name: "Lilla toan\0\0\0\0\0\0\0\0\0\0 Mode raw"
```

Temperature sensor
```
state_class: measurement
unit_of_measurement: °C
device_class: temperature
friendly_name: "Lilla toan\0\0\0\0\0\0\0\0\0\0 Temperature"
```

**Comment by tedenda** [comment]
The NULL characters is in the device.name from the device, so next step is to "clean" the name:

def get_name(device: FreshIntelliVent) -> str:
    """Generate name with identifier for device."""
    hexify = lambda s: [hex(ord(i)) for i in list(str(s))]
    _LOGGER.debug("RAW name: %s", hexify(device.name))
    return f"{device.manufacturer} {device.name}"

2022-12-30 14:17:06.068 DEBUG (MainThread) [custom_components.fresh_intellivent_sky.config_flow] RAW name: ['0x4c', '0x69', '0x6c', '0x6c', '0x61', '0x20', '0x74', '0x6f', '0x61', '0x6e', '0x0', '0x0', '0x0', '0x0', '0x0', '0x0', '0x0', '0x0', '0x0', '0x0']

**Comment by tedenda** [comment]
After a bit of research, the name is clear of rubbish.

2022-12-30 15:49:31.408 DEBUG (MainThread) [custom_components.fresh_intellivent_sky.config_flow] cleanName: ['0x4c', '0x69', '0x6c', '0x6c', '0x61', '0x20', '0x74', '0x6f', '0x61', '0x6e']

I have to clean up the code now and make a PR

**Comment by tedenda** [comment]
I opened an issue on the library instead:
https://github.com/LaStrada/pyfreshintellivent/issues/25

**Comment by tedenda** [comment]
Fixed in latest release 0.1.2
